src/test/java/seedu/storemando/model/item/ExpiryDateTest.java

```diff
@@ -56,11 +62,53 @@ public void isValidExpiryDate() {
         assertFalse(ExpiryDate.isValidExpiryDate("@2010-07-10")); // expiryDate starts with symbol
 
 
-        // valid expiryDate
-        assertTrue(ExpiryDate.isValidExpiryDate("2020-10-10"));
-        assertTrue(ExpiryDate.isValidExpiryDate("2020-01-01"));
+        // invalid expiryDate DD below boundary and above boundary
+        assertFalse(ExpiryDate.isValidExpiryDate("0000-01-32"));
```

Comment:
I think some of these test cases are too repetitive, I understand that it is to test for boundaries but i don't think this many test cases are needed. Maybe just boundary for 1/2 months etc. Same applies to other checks in this file :)

Reply:
Noted and I have fixed it accordingly. Thanks!